src/managers/GuildApplicationCommandManager.js

```diff
@@ -119,6 +120,90 @@ class GuildApplicationCommandManager extends ApplicationCommandManager {
     );
   }
 
+  /**
+   * Add permissions to a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {ApplicationCommandPermissionData[]} permissions The permissions to add to this command
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Block a role from the command permissions
+   * guild.commands.addPermissions('123456789012345678', [
+   *   {
+   *     id: '876543211234567890',
+   *     type: 'ROLE',
+   *     permission: false
+   *   },
+   * ])
+   *   .then(console.log)
+   *   .catch(console.error);
+   */
+  async addPermissions(command, permissions) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    const perms = await this.fetchPermissions(id).catch(err => {
+      if (err.httpStatus !== 404) throw err;
+      return [];
+    });
+
+    const newPermissions = permissions.slice();
+    for (const perm of perms) {
+      if (!newPermissions.some(x => x.id === perm.id)) {
+        newPermissions.push(perm);
+      }
+    }
+
+    return this.setPermissions(id, newPermissions);
+  }
+
+  /**
+   * Remove permissions from a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {UserResolvable | RoleResolvable | UserResolvable[] | RoleResolvable[]} userOrRole The user(s) and role(s)
+   * to remove from the command permissions
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Remove a user permission from this command
+   * guild.commands.addPermissions('123456789012345678', '876543210123456789')
+   *   .then(console.log)
+   *   .catch(console.error);
+   * @example
+   * // Remove multiple roles from this command
+   * guild.commands.removePermissions('123456789012345678', [ '876543210123456789', '765432101234567890' ])
+   *    .then(console.log)
+   *    .catch(console.error);
+   */
+  async removePermissions(command, userOrRole) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    let permissions = await this.fetchPermissions(id).catch(err => {
+      if (err.httpStatus !== 404) throw err;
+      return [];
+    });
+
+    if (Array.isArray(userOrRole)) {
+      const ids = userOrRole.map(x => {
+        if (!this.client.users.resolveID(x) && !this.guild.roles.resolveID(x)) {
+          throw new TypeError('INVALID_ELEMENT', 'Array', 'userOrRole', x);
+        }
+        return this.client.users.resolveID(x) ?? this.guild.roles.resolveID(x);
+      });
+
+      permissions = permissions.filter(perm => !ids.includes(perm.id));
+    } else {
+      if (typeof userOrRole !== 'string') {
+        userOrRole = this.client.users.resolveID(userOrRole) && !this.guild.roles.resolveID(userOrRole);
```

Comment:
```suggestion
        userOrRole = this.client.users.resolveID(userOrRole) ?? this.guild.roles.resolveID(userOrRole);
```

Also, the typeof check can be a little useless, since `resolveID` short-circuits them.